python/lsst/pipe/tasks/calibrateImage.py

```diff
@@ -949,6 +949,7 @@ def _apply_photometry(self, exposure, background):
                                        photo_calib.getCalibrationErr(),
                                        bbox=exposure.getBBox())
         exposure.setPhotoCalib(identity)
+        exposure.metadata["BUNIT"] = "nJy"
```

Comment:
Although I'd much rather have this done within PhotoCalib itself, I guess it's not the worst thing to do it here, even better if we also do it in `reprocessVisitImage`, and the diffim and coadd code.